recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://pypi.io/packages/source/m/mtspec/mtspec-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy
+  run:
+    - python
+    - libgfortran
+    - numpy
+
+test:
+  requires:
+    - flake8
```

Comment:
Another way to do this is to pin `flake8` to a version that is known to work. I tested `flake8 2.5.1` here and everything worked smoothly.